pydoctor/templatewriter/summary.py

```diff
@@ -160,9 +167,12 @@ def heading(self, request, tag):
         return tag.clear()("Class Hierarchy")
 
 
-class LetterElement(Element):
+class LetterElement(BaseElement):
+
+    filename = ''
```

Comment:
Maybe this is another hint that `LetterElement` shouldn't actually be a `BaseElement`.